ui/src/style/modules/_variables.scss

```diff
@@ -9,6 +9,15 @@ $z--notifications: 9999;
 $z--overlays: 9990;
 $z--drag-n-drop: 5000;
 
+$dash-graph-border-z: 4;
+$dash-graph-resizer-z: 5;
+$dash-graph-header-z: 2;
+$dash-graph-context-z: 5;
+$dash-graph-note-z: 7;
+$dash-graph-context-expanded-z: 20;
+$dash-graph-interacting-z: 21;
+$date-picker-z: 8;
```

Comment:
Nice! These seem all `$dash-graph-*` related. We could have some breathing room around it to separate it from the `dash-graph` indexes

Reply:
Good idea! I added a space